src/pygeon/filters/convert_from_pp.py

```diff
@@ -63,24 +63,25 @@ def convert_from_pp(
     return obj
 
 
-def as_mdg(sd: pp.MixedDimensionalGrid | pp.Grid) -> pp.MixedDimensionalGrid:
+def as_mdg(obj: pp.MixedDimensionalGrid | pp.Grid) -> pg.MixedDimensionalGrid:
```

Comment:
Do we want to cover the case of pg as input? I mean, if I create an octagon grid and I want to make it as a mixed-dimensional then I might have some problems.. I know, the function
```mdg = pp.meshing.subdomains_to_mdg([[obj]])```
is only a pp function, not sure what happen if we give a pg Grid there (apart a complain from mypy)